### ISSUE TITLE
Have port for 7.5 IDA? 

### ISSUE THREAD

**Issue by fatrolls** [alexander-pick/patchdiff2_ida6#1]
I can get this to work in IDA 7.0 but no 7.5 have port for this?

**Comment by alexander-pick** [comment]
I haven't updated this for ages, if it compiles for 7.0, 7.5 shouldn't be to hard to reach. The project is nearly 10 yrs old now, I wasn't even aware it is interesting for anyone anymore. Did you try BinDiff? It is pretty much state of the art today and since google bought them it is free.

**Comment by fatrolls** [comment]
Yes I also wrote a support issue on Google's BinDiff6 as it gives me a error when diffing 2 idb's I use Windows 7 and it's only for Windows 8.1+.. I managed to install it in windows 7 by editing the installer and it didn't work good.

IDA 7.0 was still back compatible with old versions. Starting at IDA 7.4

https://www.hex-rays.com/products/ida/support/ida74_idapython_no_bc695_porting_guide.shtml
https://www.hex-rays.com/products/ida/news/7_0/docs/api70_porting_guide/
